docs/modules/clients/pages/java.adoc

```diff
@@ -1,19 +1,64 @@
 = Java Client
 :page-api-reference: https://docs.hazelcast.org/docs/{page-latest-supported-java-client}/javadoc
-:url-cloud-signup: https://cloud.hazelcast.com/sign-up
-:page-toclevels: 3 
+:page-toclevels: 1
+:description: Hazelcast provides a {java-client} within the standard distribution you can start using right away, and also a lightweight {java-client-new} that is available in Beta.
 [[java-client]]
 
-TIP: For the latest Java API documentation, see https://docs.hazelcast.org/docs/{page-latest-supported-java-client}/javadoc[Hazelcast Java Client docs].
+// check redirects
 
-To get started, include the `hazelcast.jar` dependency in your classpath. Once included, you can start using this client as if
-you are using the Hazelcast API. The differences are discussed in the below sections.
+== Overview
 
-NOTE: If you have a Hazelcast {enterprise-product-name} license, you do not need to set the license key in your Hazelcast Java clients to use the xref:getting-started:editions.adoc#features-in-hazelcast-enterprise[{enterprise-product-name} features] - setting it on the member side is enough. In this case, you only need to include the `hazelcast-enterprise.jar` dependency in your classpath.
+// client options
+Hazelcast provides a native {java-client} which you can use to connect to a Hazelcast cluster. `hazelcast.jar` is bundled in the Hazelcast standard package, so just add `hazelcast.jar` to your classpath and you can start using this client as if you are using the Hazelcast API.
```

Comment:
```suggestion
Hazelcast provides a platform {java-client} which you can use to connect to a Hazelcast cluster. `hazelcast.jar` is bundled in the Hazelcast standard package, so just add `hazelcast.jar` to your classpath and you can start using this client as if you are using the Hazelcast API.
```
I am not sure if native is the terminology we want to use for existing client. We called it as Platform client. We also have native(HD) memory feature which may cause native client to be misunderstood.

Reply:
Thanks, removed (I thought it might be controversial / inappropriate and it's fine without it)